client/src/components/ImageScrubber/ImageScrubber.js

```diff
@@ -123,11 +123,9 @@ class ImageScrubber extends Component {
             >
           <div className={classes.wrapper}>
             {this.props.treesArray.map(tree => {
-                if (tree.imageUrl) {
```

Comment:
@misterinterrupt Here was the cause of for issue with the `TreeImageCards` not appearing. Many of the cards that are being return by the api do not have value for the `imageUrl` property on each `tree`. 

This condition prevents it from loading onto the page.